src/pages/in/like/_id/index.vue

```diff
@@ -213,7 +213,11 @@ export default {
               && this.$refs.likeButton
             ) {
               // Click the LikeButton directly for clicking effect
-              this.$refs.likeButton.onClick();
+              this.$nextTick(() => {
+                if (this.$refs.likeButton) {
+                  this.$refs.likeButton.onClick();
+                }
+              });
```

Comment:
Suppose the existing condition can guard this case, and `isCreator` should be set before `hasUpdateUserSignInStatus`. I think the bug is due to somehow `hasUpdateUserSignInStatus` is set to `true` before setting `isCreator`, which can deduce `isCreator` is always `false`. I think wrapping the entire condition inside `$nextTick()` is better

Reply:
I think the bug reporter might have hit `this.isCreator && this.isSubscribed` case since he is a civic liker
then somehow in the click logic, the computed properties are not updated yet, causing this bug